docs/examples/nodejs-read-versioned-layer.md

```diff
@@ -117,73 +120,73 @@ const context = new DataStoreContext({
 
 ## VersionedLayerClient
 
-Therefore, when you have context, you can get catalog clients for different catalogs and read the information.
+Therefore, when you have settings, you can get catalog clients for different catalogs and read the information.
 You can get more information about Version Layer following the [link](https://developer.here.com/olp/documentation/data-api/data_dev_guide/rest/publishing-data-versioned.html).
-To create VersionedLayerClient, run:
-
+To create `VersionedLayerClient`, run:
 
 ```typescript
-import { UserAuth } from "@here/olp-sdk-authentification";
-import { DataStoreContext, VersionedLayerClient } from "@here/olp-sdk-dataservice-read";
-import * as utils from "@here/olp-sdk-dataservice-read";
+//  Client for "protobuf-example-berlin-v1" layer from "sensor-data-sensoris-versioned-example" catalog
+const versionClient = await new VersionedLayerClient(
+    HRN.fromString("hrn:here:data:::sensor-data-sensoris-versioned-example"),
```

Comment:
please, don't use the real hrn, datahandles, etc ..

Reply:
Removed